engines/query-sparql/test/QuerySparql-test.ts

```diff
@@ -775,6 +775,54 @@ SELECT ?obsId {
 `, context)))).resolves.toHaveLength(1);
       });
     });
+
+    describe('commutative count', () => {
+      it('should count commutatively', async() => {
+        const context: QueryStringContext = {
+          sources: [
+            {
+              type: 'serialized',
+              value: `
+              @prefix ex: <http://example.org/> .
+            
+              <http://foo.org/id/graph/foo> {
+                <http://foo.org/id/object/foo>
+                  ex:foo ex:Foo ;
+                  ex:bar [] ;
+                  ex:baz "baz1", "baz2", "baz3" .
+              }
+              `,
+              mediaType: 'text/turtle',
+              baseIRI: 'http://example.org/',
+            },
+          ],
+        };
+
+        const received = (await arrayifyStream(await engine.queryBindings(`
+        SELECT (COUNT(DISTINCT ?s) AS ?subjects) (COUNT(DISTINCT ?o) as ?objects) ?p
+        FROM <http://foo.org/id/graph/foo>
+        {
+          ?s ?p ?o .
+        }
+        GROUP BY ?p
+        `, context))).sort((a, b) => a - b);
```

Comment:
Does this sort do anything? a and b will be bindings objects I suspect?
(also below)

Reply:
I meant to use the default sorting function which converts to string and compares strings. I've updated it.

Reply:
Thank you for spotting my mistake, I meant to use the default algorithm for '.sort()' which converts to strings but the linter required an explicit sort algorithm to be passed. Now I'm not sure: should I leave out the sort (which succeeds) or explicitly convert to string and sort the strings?